core/src/main/java/com/linecorp/armeria/server/annotation/ByteArrayRequestConverterFunction.java

```diff
@@ -36,16 +34,10 @@
      */
     @Override
     public Object convertRequest(ServiceRequestContext ctx, AggregatedHttpMessage request,
-                                 Class<?> expectedResultType) throws Exception {
+                                 Class<?> expectedResultType) {
```

Comment:
We should not remove the `throws` clause because a user may extend this class and override this method.